src/imap_mag/client/sdcDataAccess.py

```diff
@@ -97,45 +99,16 @@ def get_file_path(**options: Unpack[FileOptions]) -> tuple[str, str]:
         return (science_file.filename, science_file.construct_path())
 
     def upload(self, file_name: str) -> None:
-        """Upload data to imap-data-access."""
-
         logging.debug(f"Uploading {file_name} to imap-data-access.")
 
         try:
             imap_data_access.upload(file_name)
         except imap_data_access.io.IMAPDataAccessError as e:
             logging.warn(f"Upload failed: {e}")
```

Comment:
This probably should not fail in this way as it is does not propagate the exception